### ISSUE TITLE
📝No install instructions for Linux

### ISSUE THREAD

**Issue by jackgray-nycbs** [cloudflare/cloudflared#1347]
The only resource I could find to install cloudflared on linux is in a tutorial for creating a local tunnel. The repository does not find a compatible version for Ubuntu 24 LTS. 

The Linux download link in the README of this repository returns 404.

**Comment by cyanidium** [comment]
Linux downloads are listed [here](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/downloads/). On Ubuntu you'll want the .deb file or to follow the instructions in the linux sectuin about adding the Cloudflare Package Repository.

**Comment by houtianze** [comment]
The line `E: The repository 'https://pkg.cloudflare.com/cloudflared noble Release' does not have a Release file.` indicates that the cloudflared repository doesn't support Ubuntu `24.04 LTS noble` yet, as a workaround I changed to use the previous Ubuntu LTS code name `jammy` instead for `/etc/apt/sources.list.d/cloudflared.list`:
`deb [signed-by=/etc/apt/keyrings/cloudflare-main.gpg] https://pkg.cloudflare.com/cloudflared jammy main`

On the other hand, could the repository maintainer for for cloudflared please help to update the repository to support Ubuntu `noble` release?

**Comment by JokerQyou** [comment]
Yes you can workaround by using the deb file but you lose the ability to do auto-upgrade via `apt`. I'd suggest cloudflared repository maintainers to look into this issue, since Ubuntu 24.04 is being adopted as the next LTS version.
